.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
+
+[*]
+indent_size = 4
+end_of_line = lf
+
+[*.{c,h,cxx,am,m4}]
+indent_style = tab
```

Comment:
Yeah, thats what I mean.

But, I guess limiting the indent_type=tab to only the filetypes that need it is probably useful when used for editing files.  Wouldn't want Python scripts to get tabs in them would we :)